packages/venia-ui/lib/components/OrderHistoryPage/collapsedImageGallery.js

```diff
@@ -1,33 +1,20 @@
 import React, { useMemo } from 'react';
-import { useCollapsedImageGallery } from '@magento/peregrine/lib/talons/OrderHistoryPage/useCollapsedImageGallery';
+import { arrayOf, object, shape, string, number } from 'prop-types';
 
 import { mergeClasses } from '../../classify';
-import defaultClasses from './collapsedImageGallery.css';
-import imageGalleryOperations from './collapsedImageGallery.gql';
 import Image from '../Image';
-import { arrayOf, object, shape, string } from 'prop-types';
 
-const DISPLAY_COUNT = 4;
+import defaultClasses from './collapsedImageGallery.css';
 
 const CollapsedImageGallery = props => {
-    const { items } = props;
-
-    const talonProps = useCollapsedImageGallery({
-        imageCount: DISPLAY_COUNT,
-        items,
-        ...imageGalleryOperations
-    });
-    const { imageData } = talonProps;
+    const { displayCount, items, totalItemsCount } = props;
 
     const classes = mergeClasses(defaultClasses, props.classes);
-    const remainderCount = items.length - DISPLAY_COUNT;
+    const remainderCount = totalItemsCount - displayCount;
```

Comment:
`totalItemsCount` can just be derived from the `items.length`. Is there a reason it is now a prop? Similar question for the passing of `displayCount`.